src/main/scala/org/broadinstitute/dsde/firecloud/utils/PerformanceLogging.scala

```diff
@@ -0,0 +1,13 @@
+package org.broadinstitute.dsde.firecloud.utils
+
+import com.typesafe.scalalogging.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+trait PerformanceLogging {
+
+  val perfLogger: Logger = Logger(LoggerFactory.getLogger("PerformanceLogging"))
+
+  def perfmsg(caller: String, time: Long) =              s"[$caller] took [$time]"
```

Comment:
Why brackets around `$time`?

Reply:
in case we ever wanted to machine-parse these in the future, I wanted an easy-to-delineate format.